higgstautau/lephad/filters.py

```diff
@@ -0,0 +1,439 @@
+from rootpy.tree.filtering import EventFilter
+
+from math import *
+import math
+
+from .. import utils
+from ..units import GeV
+from .. import datasets
+from . import track_counting
+from .. import tauid
+from ..tauid import IDLOOSE, IDMEDIUM, IDTIGHT
+from . import log; log = log[__name__]
+
+
+class LeptonSelection(EventFilter):
+    def __init__(self,min_leps,**kwargs):
+        super(LeptonSelection,self).__init__(**kwargs)
+        self.min_leps = min_leps
+        #require lepton candidates
+    def passes(self, event):
+        return len(event.electrons)+len(event.muons) >=self.min_leps
```

Comment:
@vincecr0ft len(event.electrons) + len(event.muons) >= self.min_leps (note the blanck space between the operations)